src/Common/Entity/Transaction.php

```diff
@@ -202,31 +202,35 @@ public function setProducts($products)
         $this->products = $container;
     }
 
+    public function getTransactionSource(): string
+    {
+        return $this->transaction_source;
```

Comment:
Unless there's a default value set for the transaction_source property, I believe using this method could throw an exception.

Reply:
There is, we default to "CLIENT" and the field is not NULL.

That said, I did ask on monday what yer thoughts on that were, so I figured some discussion was warranted.

Reply:
After some discussion, we'll be swapping "client" to "none"